src/baseEntity.js

```diff
@@ -90,9 +84,7 @@ class BaseEntity {
   static fromJSON(json, options = { allowExtraKeys: false }) {
 
     function parse(type, value) {
-      if (value === undefined) return undefined
```

Comment:
Why did you removed this?

Reply:
Because this line is uncovered by the tests since it never ocurrs;
If you look at line *119* you'll see that before reaching the line that executes the function `parse` it will jump to the next iteration if the value is undefined and since this function is only used in this context  a undefined value will never enter the `parse` function making the deleted line unecessary
```
 if (!(field.constructor.name === "Field" && !checker.isFunction(field.definition) && data[key] !== undefined)) continue
```


What u guys think?